### ISSUE TITLE
how to train my own model

### ISSUE THREAD

**Issue by virus1987** [shuyucool/U-net-segmentation#1]
i used your code to change a little for study,please help me ~many thanks

i want to use model.save（）to save the model after my self trainning and then to use  load_model() to call the model for test 
but the result is wrong. please help me to check if save model is error
here is my code
history = model.fit(x=imgs_train, y=imgs_mask_train ,batch_size=1, epochs=2, verbose=1, callbacks=None,
            validation_split=0.0, validation_data=None, shuffle=True,
            class_weight=None, sample_weight=None, initial_epoch=0)
model.save('../data_set/my_unet.hdf5')
......
when test and predict:
 model = self.get_unet()
        print("got unet")
        model= load_model('../data_set/my_unet.hdf5')
        print('predict test data')
        imgs_mask_test = model.predict(imgs_test, batch_size=1, verbose=1)
        np.save('../data_set/imgs_mask_test.npy', imgs_mask_test)
